src/infra/monitoring/queries/stamp/BackgroundProcessorHealthStatus.kql

```diff
@@ -7,7 +8,7 @@ let Thresholds=datatable(MetricName: string, YellowThreshold: double, RedThresho
     "avgProcessingTime", 100, 200
     ];
 //
-let timespanStart= todatetime(format_datetime( todatetime(toscalar(AppRequests| summarize min(TimeGenerated))), 'yyyy-MM-dd HH:ss'));
+let timespanStart = _timespanStart;//todatetime(format_datetime( todatetime(toscalar(AppRequests| summarize min(TimeGenerated))), 'yyyy-MM-dd HH:ss'));
```

Comment:
do you foresee ever switching back to the old code? if not, I would just remove it

Reply:
Yes, I hope we can move back to that soon. Would be great if we can control the timespan elsewhere.